### ISSUE TITLE
VSCode settngs.json 

### ISSUE THREAD

**Issue by gbowne1** [taotnpwaft/projects#14]
If this file is used as Userspace settings.json instead of Workspace, VSCode:

Has problems with C and HTML 
- [ ] pip runs at startup of VSCode
- [ ] C files .c and .h get recognized as either G-Code or C++
- [ ] .html files get recognized as Django Template
- [ ] .jsx or javascriptreact files get recognized as Babel Javascript instead of javascriptreact.

Python Language specific settings need to be completed.

Can a jsconfig.json be in the .vscode folder? I could not find good references for jsconfig.json other that there was a reference that VSCode uses it to configure itself for Javascript.

Also the VSCode workspace file is basically boilerplate.  Ive never seen any project use it let alone good settings.

@taotnpwaft/members 
@Hitblast

**Comment by gbowne1** [comment]
I think I figured out the next update to settings that should do wonders for VS Code.

I also need to add keybindings.json file for VS Code but I do not know of any good keybindings.

I also think we should provide a much generic .vs folder with better files and for Visual Studio 2019/2022.  @ScottLilly @idormain

The idea being is that if a person/user is new to an IDE or editor we want to be a source for good setting for that esitor or IDEs other than theme, colors, font etc so that sll the user has to do is do a small amount of theme, font, color, etc customization.

**Comment by idormain** [comment]
Hiya I’ll be using Visual Studio 2022 or Rider as my IDESent from my iPhoneOn 23 Oct 2022, at 04:04, Gregory Bowne ***@***.***> wrote:﻿
I think I figured out the next update to settings that should do wonders for VS Code.
I also need to add keybindings.json file for VS Code but I do not know of any good keybindings.
I also think we should provide a much generic .vs folder with better files and for Visual Studio 2019/2022.  @ScottLilly @idormain
The idea being is that if a person/user is new to an IDE or editor we want to be a source for good setting for that esitor or IDEs other than theme, colors, font etc so that sll the user has to do is do a small amount of theme, font, color, etc customization.

**Comment by gbowne1** [comment]
@idormain

Does Rider have its own .config folder like .vscode or .vs or .idea?

I think Visual Studio will intelligently pick up on .vscode folder if theres no .vs folder.

I also found that some browsers like Firefox will pick up files like user.json.

I would like to make it easier for us and anyone else to get set up with minimal config.

That way someome doing say NodeJS dev has a quick config they can replace the default with and get going.

**Comment by idormain** [comment]
Rider can use the config provided Sent from my iPhoneOn 23 Oct 2022, at 06:50, Gregory Bowne ***@***.***> wrote:﻿
@idormain
Does Rider have its own .config folder like .vscode or .vs or .idea?
I think Visual Studio will intelligently pick up on .vscode folder if theres no .vs folder.
I also found that some browsers like Firefox will pick up files like user.json.
I would like to make it easier for us and anyone else to get set up with minimal config.
That way someome doing say NodeJS dev has a quick config they can replace the default with and get going.

**Comment by gbowne1** [comment]
Ahh. Great. I was wondering what the format was or if the  config was in a particular folder.

**Comment by idormain** [comment]
Do you have database schema available? Or just use the Models?Sent from my iPhoneOn 23 Oct 2022, at 10:38, Gregory Bowne ***@***.***> wrote:﻿
Ahh. Great. I was wondering what the format was or if the  config was in a particular folder.

**Comment by gbowne1** [comment]
@idormain 

Yes I do.  Its in Dropbox, and Ive added the unconverted .zip to the App Noes discussion of the ERP.  Let me know if you need more or anything else. I have the ERD too.
